### ISSUE TITLE
valid-jsdoc fails on functions that use ES6 destructuring + default value in signature

### ISSUE THREAD

**Issue by SpainTrain** [eslint/eslint#6101]
Related to #2051.  Destructuring works alone, but if using destructuring with a default value, the same valid-jsdoc error is emitted as in #2051.  Using the example from #2051:

```js
/**
 * @param {number} a
 * @param {number} b
 * @param {number} c
 * @return {number}
 */
const test = function({ a, b, c } = {a: 0, b: 0, c: 0}) {
  return a + b + c;
};
module.exports = test;
```

```
error  Expected JSDoc for 'undefined' but found 'a'  valid-jsdoc
```

- eslint 2.9.0
- babel-eslint 6.0.4

**Comment by mysticatea** [comment]
Thank you for this issue. But closing as duplicate of #6097. Please track that issue.
